sources/vega/chain_events.proto

```diff
@@ -67,6 +67,30 @@ message ERC20Withdrawal {
   string reference_nonce = 3;
 }
 
+// A new signer added to the ERC20 bridge
+message ERC20SignerAdded {
+  // The ethereum address of the new signer
+  string new_signer = 1;
```

Comment:
Minor: `string signer_address` feels a better name, so you _know_ the field is an eth-address. And we don't really need `new` in there since its implied by `Added` in `ERC20SignerAdded`.

Same goes for `old_signer` in `ERC20SignerRemoved`.

Reply:
Yea I agree, although this is named like so just to match the actual contract events.